### DISCUSSION TITLE
A simple question (maybe).

### DISCUSSION THREAD

**Discussion by Jessebob42** [EDCD/EDDI#2199] [EDDI Standalone Q&A]
I've been using EDDI for years.  It's running 99% of the time when I'm playing Elite Dangerous.  But then there are different addons out there I've been looking at and I'm wondering if I'm just duplicating my efforts at providing data.
I plan on keeping EDDI. (I also upload my .logs on EDSM).
1. Will running ED Market Connector add anything to data collection or will EDDI do the same thing.
2. Does EDDI get all system market information if I scan the Nav Beacon? Or do you need to visit the port?
3. Future Feature Desire: A single Remove All button on the Crime Monitor tab. 
4. 👍🏻🍻
5. Are there any other addons that could provide data that isn't provided by EDDI?

**Comment by Tkael** [comment]
1. It's hard for me to answer this question because I don't know what plugins you are using (or looking at using) with EDMC. EDDI sends data to EDDN, EDSM, and Inara which is enough to satisfy most users ;-). You can certainly run both EDMC and EDDI together but it is best to configure only one tool to send to each service to avoid duplicating effort.

2. Not at this time. You'd have to visit the port.

3. We acknowledge that the Crime Monitor needs some work - it tends to collect data and not clear it when it should. I'm hopeful that we'll be able to make improvements that will diminish the need for a "Remove All" button.

4. 🍻

5. What kind of data are you looking for?

**Comment by Jessebob42** [comment]
I don’t need the raw data. I just want to send you complete data in the most efficient way.  Avoiding duplication of effort. 

Did I ask a #5? If I were to ask a #5 it would now be: Does the Nav Beacon (next to the primary star) does that update market data or is a visit to the station required?

Regards 🍻
Jesse Eissens (Commander Zund)

**Reply by Tkael** [reply]
No. Scanning the nav beacon provides us with a scan of each body in the system but does not update any market information.
